eks-anywhere-common/Testers/Solo.io/solo-istiod-testJob.yaml

```diff
@@ -0,0 +1,66 @@
+apiVersion: batch/v1
+kind: CronJob
+metadata:
+  name: solo-istiod-healthtest
+  namespace: istio-system
+spec:
+  schedule: "*/10 * * * *"
+  jobTemplate:
+    spec:
+      template:
+        spec:
+          serviceAccountName: tester
+          containers:
+            - name: test-solo-istiod
+              image: alpine/k8s:1.26.9
+              command:
+                - /bin/sh
+              args:
+                - -c
+                - >-
+                  istiod_pod_state=$(kubectl get pods -n istio-system -l app=istiod -o json | jq -r .items[].status.containerStatuses[].ready);
+                  echo Getting current istiod state.;                  
```

Comment:
Please check our [Functional requirements](https://aws-samples.github.io/eks-anywhere-addons/#functional-job-requirements) for test job. Container Status does not align to our requirements of functional job.

Reply:
Thanks @elamaran11 - quick question if the test is replaced with curl call similar to SUSE [here](https://github.com/aws-samples/eks-anywhere-addons/blob/main/eks-anywhere-common/Testers/SUSE/neuvector-testJob.yaml) will it satisfy?

Reply:
@elamaran11 - please let us know if modified testJob is acceptable. Thank you!

Reply:
@PetrMc Unfortunately this is not a good example. It was early days before we created those requirements. check other partners like Kong, NewRelic, Pulumi, Nirmata with examples which satisfies all requirements. Please go over the functional job reqs you will get an idea.

Reply:
@elamaran11 - please let me know if the new re-written functional test will satisfy. Thank you and apologies for multiple commits!